Userland/mount.cpp

```diff
@@ -0,0 +1,47 @@
+#include <LibCore/CArgsParser.h>
+#include <sys/stat.h>
+#include <stdio.h>
+#include <unistd.h>
+
+// This version of 'mount' must have the following arguments
+// 'mount <device_path> <mount_point>
+// It can currently only mount _physical_ devices found in /dev
+//
+// Currently, it is only possible to mount ext2 devices. Sorry! :^)
+int main(int argc, char** argv)
+{
+    CArgsParser args_parser("mount");
+    args_parser.add_arg("devname", "device path");
+    args_parser.add_arg("mountpoint", "mount point");
+    CArgsParserResult args = args_parser.parse(argc, argv);
+
+    if (argc == 3) {
+        // Let's use lstat so we can convert devname into a major/minor device pair!
```

Comment:
This would be better to do in the kernel. Device major/minor numbers are a kernel detail, not something we should worry about in userspace.

Reply:
Hmm, what includes do I use for this??? I assume I'd be doing this bit of code in `Process::sys$mount` yeah?

Reply:
Yeah, you can look how `Process::sys$stat` is implemented :)